contrib/ansible/Getting-Started.md

```diff
@@ -0,0 +1,157 @@
+
+
+# Kubernetes Cluster with Containerd and Cri-Containerd
+
+In this document we aim to provide simple steps to bring up a Kubernetes cluster using ansible and kubeadm tools. 
+
+### Prerequisites:
+- **OS**: Ubuntu 16.04 (Other OS installations will be validated and updated)
+- **Python**: 2.7+
+- **Ansible**: 2.3+
+
+## Step 0:
+-  Install Ansible on any one of the nodes in the candidate cluster or an external machine which will be used to provision the cluster. The install instructions can be found [here](http://docs.ansible.com/ansible/latest/intro_installation.html). The playbooks in the repo need ansible version 2.3 or later.
+-  Create a hosts file and include the Routable IPs of the nodes that need to be provisioned by ansible.
+```
+$ cat hosts
+172.31.7.230
+172.31.13.159
+172.31.1.227
+```
+-  Setup passwordless SSH access from ansible control machine to all the hosts in the hosts file. The instructions can be found in [here](http://www.linuxproblem.org/art_9.html)
+
+## Step 1:
+At this pointthe ansible playbook should be able to ssh into all the machines in the hosts file and start installing the playbook
```

Comment:
/pointthe/point the/

Reply:
point[,] the
/all//

Reply:
[the] playbook[.]

Reply:
At this point, the ansible playbook should be able to ssh into the machines in the hosts file. 

Install the Ansible playbook: